Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -167,6 +183,25 @@ public class HTTPNetworkTransport {
     
     return task
   }
+
+  private func handleGraphQLErrorsOrComplete<Operation>(operation: Operation,
```

Comment:
I think this needs to include a `files` parameter in case an operation with files needs to get retried

Reply:
Just saw the `handleErrorOrRetry` also misses that? 
Want me to add here or in a new PR?

Reply:
Add here - I think the reason it doesn't handle it there is that it's more specifically looking at retries for APQ stuff rather than in general, and APQ can only be done with Queries, not mutations.